File: ModdedBE-Android/app/src/main/java/net/listerily/moddedbe/MainActivity.java
package net.listerily.moddedbe;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onStartGameClicked(View view) {
        startActivity(new Intent(this, InitializingActivity.class));
        finish();
    }

    public void onMenuClicked(View view) {
    }
}

File: ModdedBE-Android/app/src/main/java/net/listerily/moddedbe/MainActivity.kt
package net.listerily.moddedbe

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onStartGameClicked(view: View) {
        startActivity(Intent(this, InitializingActivity::class.java))
        finish()
    }

    fun onMenuClicked(view: View) {
        startActivity(Intent(this, OptionsActivity::class.java))
    }
}